libraries/lib-numeric-formats/NumericConverter.h

```diff
@@ -45,9 +45,6 @@ struct NumericField
    int range; // then take modulo this
    int digits;
    int pos;   // Index of this field in the ValueString
-   int fieldX; // x-position of the field on-screen
-   int fieldW; // width of the field on-screen
-   int labelX; // x-position of the label on-screen
```

Comment:
This commit is good.

I would now make it better:

`NumericConverter::mFields` should be private, and a protected member function gives `NumericTextCtrl` only the `const` access to the array that it requires.